backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -135,13 +135,13 @@ private async void OnIsarRobotInfo(object? sender, MqttReceivedArgs mqttArgs)
 
                 List<string> updatedFields = [];
 
-                if (isarRobotInfo.VideoStreamQueries is not null) { UpdateVideoStreamsIfChanged(isarRobotInfo.VideoStreamQueries, ref robot, ref updatedFields); }
-                if (isarRobotInfo.Host is not null) { UpdateHostIfChanged(isarRobotInfo.Host, ref robot, ref updatedFields); }
+                if (isarRobotInfo.VideoStreamQueries is not null) UpdateVideoStreamsIfChanged(isarRobotInfo.VideoStreamQueries, ref robot, ref updatedFields);
```

Comment:
It could be good to also apply these checks and update strategy in the Update endpoint in the Robot controller, as this currently does not work. We can do this in another PR though, and we would need a new query class since passing a Robot object to the endpoint is not ideal.

Reply:
I have updated the mentioned endpoint in a different PR, so no worries